fusesoc/config.py

```diff
@@ -124,18 +124,13 @@ def __init__(self, path=None, file=None):
                 logger.warning(_s.format(str(e), name))
                 continue
 
-            try:
-                sync_uri = config.get(section, "sync-uri")
-            except configparser.NoOptionError:
-                # sync-uri is absent for local libraries
-                sync_uri = None
+            sync_uri = config.get(section, "sync-uri", None)
+            sync_version = config.get(section, "sync-version", None)
+            sync_type = config.get(section, "sync-type", None)
 
-            try:
-                sync_type = config.get(section, "sync-type")
-            except configparser.NoOptionError:
-                # sync-uri is absent for local libraries
-                sync_type = None
```

Comment:
I neatened these up while I was here, as it looks like python2 support was dropped (#337), lmk if I shouldn't have :)